packages/stog/stog.0.18.0/opam

```diff
@@ -11,7 +11,7 @@ build: [
   ["./configure" "--prefix" prefix]
   [make "all"]
 ]
-install: [make "install-lib" "install-share"]
+install: [make "install-lib" "install-share" "install-bin"]
 remove: ["ocamlfind" "remove" "stog"]
```

Comment:
Given that we are here, and this line is not necessary, would you mind removing it?
```suggestion
```